src/Altinn.AccessManagement.Core/Helpers/DelegationHelper.cs

```diff
@@ -164,10 +166,13 @@ public static bool TryGetDelegationParamsFromRule(Rule rule, out ResourceAttribu
                 coveredByPartyId = TryGetCoveredByPartyIdFromMatch(rule.CoveredBy, out int coveredByParty) ? coveredByParty : null;
                 coveredByUserId = TryGetCoveredByUserIdFromMatch(rule.CoveredBy, out int coveredByUser) ? coveredByUser : null;
                 delegatedByUserId = rule.DelegatedByUserId;
+                delegatedByPartyId = rule.DelegatedByPartyId;
+                delegatedDateTime = rule.DelegatedDateTime ?? DateTime.UtcNow;
```

Comment:
What if the input is not utc? If this is input from Altinn 2 is it converted to utc before setting it on the input rule?

Reply:
The datetime object just need to hold timesone info so will the db adjust for this on storing to db changed the building of json from Altinn 2 to include timezone info so nothing is lost in translation. Do not know the timezone used in AltinnPlatform but when it is sendt from Altinn II with +01:00 or +02:00 it will be translated to local handling with necesary adjustment